static/genpackage-jbrowse/app/directives.js

```diff
@@ -69,6 +66,10 @@ angular.module('jbrowse.directives', ['genjs.services', 'jbrowse.services'])
                 resolvedDefer.resolve();
                 resolvedPromise = resolvedDefer.promise;
 
+                function fileUrl(id, filename, dontEscape) {
```

Comment:
File service would make an ajax request and return the `file data`.
JBrowse handles ajax requests on its own and accepts `file urls`.

I dont think there is a way to use angular-resource to only generate url :)